src/main/java/com/ays/institution/model/dto/response/InstitutionResponse.java

```diff
@@ -13,8 +11,7 @@
 @Getter
 @Setter
 @SuperBuilder
-@EqualsAndHashCode(callSuper = true)
-public class InstitutionResponse extends BaseResponse {
```

Comment:
@emreylmaz @Rapter1990 Buradaki `BaseResponse` kısmını kaldırarak diğer endpointleri etkilemiş oluyoruz. Aslında burası çok kritik, siz etkilenen yerleri göz önüne almadan bu değişikliği yaptığınız için var olan endpointlerde de hataya sebebiyet veriyoruz. Bu akış için yeni bir sınıf oluşturmak gerekiyor. Buradaki değişiklikleri geri almamız daha iyi olur.

Reply:
Bu kısmı revise ettim